### DISCUSSION TITLE
数据质量模块当中的数据完整度SQL的一个小bug

### DISCUSSION THREAD

**Discussion by tkatt** [apache/iotdb#3982] [Show and tell]
我在使用数据完整度completeness函数时，指定窗口大小的时候，不小心把window拼成了windown，但是sql居然没报错

`IoTDB> select completeness(wtid, "windown"='1d', "downtime"="false") from root.gw.7s.654323.654323001
+------------------------+----------------------------------------------------------------------------------+
|                    Time|completeness(root.gw.7s.654323.654323001.wtid, "windown"="1d", "downtime"="false")|
+------------------------+----------------------------------------------------------------------------------+
|2020-01-01T00:00:09.000Z|                                                                0.9687289698873334|
+------------------------+----------------------------------------------------------------------------------+`

**Comment by SteveYurongSu** [comment]
@tkatt Thanks for your report! 😆

@543202718 Plz do a check. Tips for you: UDF's user input parameters such as `window` can be checked by the `UDFParameterValidator#validateRequiredAttribute()` method.
